src/callbacks/findNDI.js

```diff
@@ -0,0 +1,11 @@
+function findNDI(response) {
```

Comment:
what is NDI? Nearest driver? also please change to anonymous function

Reply:
done